searx/templates/simple/opensearch_response_rss.xml

```diff
@@ -21,34 +21,11 @@
     {% for r in results %}
     <item>
       <title>{{ r.title }}</title>
+      <type>result</type>
       <link>{{ r.url }}</link>
       <description>{{ r.content }}</description>
       {% if r.pubdate %}<pubDate>{{ r.pubdate }}</pubDate>{% endif %}
     </item>
     {% endfor %}
-    {% if answers %}
-      {% for a in answers %}
-      <item>
-        <title>{{ a }}</title>
-        <type>answer</type>
-      </item>
-      {% endfor %}
-    {% endif %}
-    {% if corrections %}
-      {% for a in corrections %}
-      <item>
-        <title>{{ a }}</title>
-        <type>correction</type>
-      </item>
-      {% endfor %}
-    {% endif %}
-    {% if suggestions %}
-      {% for a in suggestions %}
-      <item>
-        <title>{{ a }}</title>
-        <type>suggestion</type>
-      </item>
-      {% endfor %}
-    {% endif %}
```

Comment:
Thanks to this PR its the first time I see we had answers, corrections and suggestions in the result list.  Theses had been marked by a `<type>` tag .. but in the RSS 2.0 spec there does not exists such a tag .. instead of producing invallid XML, the RSS feed should only contain real results, nothing more.

Reply:
True, I agree on that.